impl/src/main/java/io/quarkiverse/kafkastreamsprocessor/impl/DefaultSinkToTopicMappingBuilder.java

```diff
@@ -44,7 +58,11 @@
  */
 @ApplicationScoped
 @Slf4j
-public class DefaultSinkToTopicMappingBuilder implements SinkToTopicMappingBuilder {
+public class DefaultSinkToTopicMappingBuilder {
```

Comment:
Since the interface has been removed, you can remove "Default" from the implementation class name. Same for the Source one.